CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v2.3.0 - 2023-12-15
+
+### Added
+
+- [#245](https://github.com/networktocode/circuit-maintenance-parser/pull/245) [#250](https://github.com/networktocode/circuit-maintenance-parser/pull/250) [#252](https://github.com/networktocode/circuit-maintenance-parser/pull/252) - feat: :sparkles: OpenAI parser by @chadell
```

Comment:
Just a question - `:sparkles:` is GitHub specific I presume? For the compiled docs I'd expect an actual unicode character or similar, or else remove it entirely?